win32ss/gdi/gdi32/objects/font.c

```diff
@@ -1978,6 +1978,7 @@ TranslateCharsetInfo(
         while (index < MAXTCIINDEX && !(*lpSrc>>index & 0x0001)) index++;
         break;
     case TCI_SRCCODEPAGE:
+        /* ciACP is also referred by IntCharSetFromCodePage() function */
```

Comment:
Actually I was thinking about adding the comment in IntCharSetFromCodePage(), not here.